OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -646,6 +647,8 @@ namespace OloEngine
 
     void Scene::OnUpdateRuntime(Timestep const ts)
     {
+        auto* perfProfiler = Application::Get().GetPerformanceProfiler();
```

Clang (14.0.6)

[warning] variable 'perfProfiler' is not initialized

(cppcoreguidelines-init-variables)